content/post/31-js-ipfs-0.26.md

```diff
@@ -0,0 +1,96 @@
+
+### The libp2p-webrtc-star multiaddrs have been fixed
+
+@diasdavid (me) and @lgierth had a [good convo and reviewed a bunch of stuff over Coffee ☕️, it was great!](https://github.com/ipfs/pm/blob/master/meeting-notes/2017-08-30--coffee-lars-david.md). During that chat, we figured that `libp2p-webrtc-star` multiaddrs have been implemented incorrectly and figured out the migration path to the correct version. 
+
+You can learn more what this endeavour involved here https://github.com/ipfs/js-ipfs/issues/981. Essentially, there are no more `/libp2p-webrtc-star/dns4/star-signal.cloud.ipfs.team/wss`, instead we use `/dns4/star-signal.cloud.ipfs.team/wss/p2p-webrtc-star` which signals the proper encapsulation you expect from a multiaddr.
+
+### New example showing how to stream video using hls.js
+
+@moshisushi developed a video streamer on top of js-ipfs and shared an example with us. You can now find that example as part of the examples set in this repo. Check https://github.com/ipfs/js-ipfs/tree/master/examples/browser-video-streaming, it is super cool 👏🏽👏🏽👏🏽👏🏽.
+
+> HLS (Apple's HTTP Live Streaming) is one of the several protocols currently available for adaptive bitrate streaming.
+
+### webcrypto-ossl was removed from the dependency tree
+
+We've purged webcrypto-ossl from the dependency tree. It was only used to generate RSA keys faster (significantly faster) but at the same time, it caused a lot of hurdles to being a native dependency. 
+
+There is an [open issue on the Node.js project to expose the RSA key generation primitive](https://github.com/nodejs/node/issues/15116), if you have a use case for it please do share it in that thread. This would enable js-ipfs to use the native crypto module, have the same (or better) perf of webcrypto-ossl and not have to deal with an external native dependency.
+
+### PubSub tutorial published
+
+@pgte published an amazing tutorial on how to use PubSub with js-ipfs and in the browser! Read it on the [IPFS Blog https://blog.ipfs.io/29-js-ipfs-pubsub](https://ipfs.io/blog29-js-ipfs-pubsub/).
+
+# 🌱 Future
+
+We have so much stuff in the pipeline, some of it almost got into this release but we eventually decided to defer it to a next release and make sure everyone would have access to the goodies in the _highlights_ section. That said, expect the following and more to come in a future release.
+
+### Circuit Relay
+
+Circuit Relay is almost here, @dryajov showed us a demo of it working on the last IPFS All hands, you can watch it on the uploaded recording in YouTube https://youtu.be/chAXj_vsR2s?t=25m01s. Since then, @dryajov @stebalian and @vyzo have been working on making sure interop is fully tested.
```

Comment:
old: "almost here, @dryajov"
new: "almost here. @dryajov"

old: "IPFS All hands, you can"
new: "IPFS All Hands, and you can" (capital H in Hands. also add 'and' to connect the phrases, or if you prefer can separate into different sentences)